ExaFunctions/CsvBulkImportFromBlobStorage.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Text;
+using Exasol.Utilities;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Exasol
+{
+    public static class CsvBulkImportFromBlobStorage
+    {
+        [FunctionName("CsvBulkImportFromBlobStorage")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                log.LogInformation("C# HTTP trigger function processed a request.");
+
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                dynamic data = RequestUtilities.DeserializeJsonBody(requestBody);
+
+                CheckParameters(data);
+                //POST body
+                //the target db connection string
+                string dbConnectionString = data?.dbconnectionstring;
+                //the target db table
+                string dbTable = data?.dbtable;
+                //the azure storage account connection string
+                string azureStorageAccountConnectionString = data?.storageaccountconnectionstring;
+                //the name of the blob storage container withing the storage account
+                string azureStorageAccountContainerName = data?.storageaccountcontainername;
+                //the filepath within the blob storage container
+                string azureStorageAccountContainerPath = data?.storageaccountcontainerpath;
+
+                var filenameList = AzureUtilities.GetAzureFilesListInBlobContainerFolder(azureStorageAccountConnectionString, azureStorageAccountContainerName);
+                var filteredFilenameList = FilterUtilities.FilterFilenameListOnDirectoryPath(filenameList, azureStorageAccountContainerPath);
+
+                string bulkloadingQuery = BulkloadingUtilities.ConstructBulkLoadingQuery(dbTable, azureStorageAccountConnectionString, azureStorageAccountContainerName, filteredFilenameList);
+                var response = QueryUtilities.RunNonQuery(dbConnectionString, bulkloadingQuery);
+                return new OkObjectResult(response);
+
+            }
+            catch (Exception e)
+            {
```

Comment:
This is just my personal preference, but I would keep this on one line to make the code more compact. And I like removing unnecessary empty lines in codes

Reply:
As we discussed, it's the default formatting./ style.